www/js/app.js

```diff
@@ -1325,83 +1399,117 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         var htmlCSS = htmlArticle.match(/<html[^>]*class\s*=\s*["']\s*([^"']+)/i);
         htmlCSS = htmlCSS ? htmlCSS[1] : '';
         
-        // Tell jQuery we're removing the iframe document: clears jQuery cache and prevents memory leaks [kiwix-js #361]
-        $('#articleContent').contents().remove();
-
         // Hide any alert box that was activated in uiUtil.displayFileDownloadAlert function
         $('#downloadAlert').hide();
 
-        var iframeArticleContent = document.getElementById('articleContent');
-        
-        iframeArticleContent.onload = function() {
-            iframeArticleContent.onload = function(){};
+        // Code below will run after we have written the new article to the articleContainer
```

Comment:
Comment is incorrect now. Should read:

`// Code below will run after we have loaded the dummy article into the articleContainer`